### ISSUE TITLE
Anyone can see all the dashboard without login

### ISSUE THREAD

**Issue by BillHoo** [Hygieia/Hygieia#3031]
Hi there, I'm setup Hygiea successfully and logged in with admin, created some dashboard then logout from admin. But, I could still see all the dashboard from the main page, is there any feature to restrict anonymous usage?

**Expected:**

When anyone visit my Hygieia main page, there should only be a Login page, nobody can see the dashboard unless they login.

**Comment by rvema** [comment]
@BillHoo Its not supported right now. We would like to view the dashboards without login and if there is a requirement for you to enforce such a feature you can checkout the code and modify per your needs. however we cannot implement that feature in the master branch.

**Comment by BillHoo** [comment]
@rvema I got your point, see the dashboard at very first place will encourage developer/excutive to check their DevOps status right? But I'm still curious how CapitalOne prevent from non-teammember to see your own dashboard, if I got the address of your Hygieia, I'm seeing your dashboard right?

I have this question because we are not that big than CapitalOne, we have limited network skills to setup firewalls or internal sub networks. Thank you very much.
